CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,136 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, caste, color, religion, or sexual
+identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the overall
+  community
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or advances of
+  any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email address,
+  without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Enforcement Responsibilities
+
+Community leaders are responsible for clarifying and enforcing our standards of
+acceptable behavior and will take appropriate and fair corrective action in
+response to any behavior that they deem inappropriate, threatening, offensive,
+or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, and will communicate reasons for moderation
+decisions when appropriate.
+
+## Scope
+
+This is the default Code of Conduct for all repositories within the `siemens`
+organization on GitHub.com.
+
+It does not apply for repositories that have their own Code of Conduct. It also
+does not apply for forks where you should follow the upstream Code of Conduct,
```

Comment:
Hm, as it's IMO implicit, shouldn't there be an explicit wording about the "direction" of forks?:
1. upstream forks, i.e., forks in `github/siemens` cloned from some upstream source
2. downstream forks, i.e., forks from `github/siemens` repos hosted in some downstream repo

Reply:
Nitpicking, but it says it doesn't apply to repos having their own CoC, then restating the same for forks, i.e., repos forked into the `github/siemens` namespace, so it's redundant? So, just saying 

> It does not apply for repositories that have their own Code of Conduct.

does include the cases as we cannot control downstream forks of `github/siemens` repos and don't overrule CoCs from repos we fork/clone and host in `github/siemens`. What about the the remaining case: an upstream repo has no CoC but is forked into `github/siemens`?

Reply:
Regarding downstream forks: we already state this only applies to projects in this namespace only, so that's clear to me it can't apply outside, but I'll try to come up with a better wording.

And for the second case I'd say the same and keep it as is - most forks are meant for contributions back upstream. It'd be a bit presumptuous to automatically apply our own CoC to those even if upstream is missing one. If a fork has a different purpose IMO best to have an explicit file there. WDYT?